esmvaltool/cmorizers/data/cmorizer.py

```diff
@@ -0,0 +1,529 @@
+        datasets = []
+        if self.datasets:
+            return self.datasets
+        for tier in tiers:
+            for dataset in os.listdir(os.path.join(self.rawobs, tier)):
+                datasets.append(dataset)
+
+        return datasets
+
+    def format_dataset(self, dataset, start, end, install):
+        """Format a single dataset.
+
+        Parameters
+        ----------
+        dataset: str
+            Dataset name
+        start: datetime
+            Start of the period to format
+        end: datetime
+            End of the period to format
+        install: bool
+            If True, automatically copies the data to the final location if
+            there is no
+        """
+        reformat_script_root = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), 'formatters',
+            'datasets', self._dataset_to_module(dataset))
+        tier = self._get_dataset_tier(dataset)
+        if tier is None:
+            logger.error('Data for %s not found', dataset)
+            return False
+
+        # in-data dir; build out-dir tree
+        in_data_dir = os.path.join(self.rawobs, tier, dataset)
+        logger.info("Input data from: %s", in_data_dir)
+        out_data_dir = os.path.join(self.output_dir, tier, dataset)
+        logger.info("Output will be written to: %s", out_data_dir)
+        if not os.path.isdir(out_data_dir):
+            os.makedirs(out_data_dir)
+
+        # all operations are done in the working dir now
+        os.chdir(out_data_dir)
+        # figure out what language the script is in
+        logger.info("Reformat script: %s", reformat_script_root)
+        if os.path.isfile(reformat_script_root + '.ncl'):
+            reformat_script = reformat_script_root + '.ncl'
+            success = self._run_ncl_script(in_data_dir, out_data_dir, dataset,
+                                           reformat_script, start, end)
+        elif os.path.isfile(reformat_script_root + '.py'):
+            success = self._run_pyt_script(in_data_dir, out_data_dir, dataset,
+                                           start, end)
+        else:
+            logger.error('Could not find formatter for %s', dataset)
+            return False
+        if not success:
+            logger.error('Foramtting failed for dataset %s', dataset)
+            return False
+        if install:
+            rootpath = self.config['rootpath']
+            target_dir = rootpath.get('OBS', rootpath['default'])[0]
+            target_dir = os.path.join(target_dir, tier, dataset)
+            if os.path.isdir(target_dir):
+                logger.info(
+                    'Automatic installation of dataset %s skipped: '
+                    'target folder %s already exists', dataset, target_dir)
+            else:
+                logger.info('Installing dataset %s in folder %s', dataset,
+                            target_dir)
+                shutil.copytree(out_data_dir, target_dir)
+        return True
```

Comment:
The documentation of the `format` and `prepare` commands says that `--install=True` would move the cmorized data from the run_dir to an OBS directory specifiec in the config-user.yml file while it actually does a copy. I would prefer to have only one version of the cmorizered data and remove them from the run_dir if  `--install=True` is used. What do you think? It would be good to make the command description consistent with the function above.

Reply:
Now it actually moves and not copies. I will later do a pass to the doc to see that it is clearer now